.travis.yml

```diff
@@ -1,19 +1,20 @@
 language: go
+os:
```

Comment:
So we run each build (even testing) on macos and linux.  Seems fine, assuming we have the build capacity on travis, licenses, etc?

Reply:
I think we're good for the time being, but agree this could/should be optimised at some point.